CHANGES.rst

```diff
@@ -1,6 +1,11 @@
 Changes
 =======
 
+Version 0.9.2 (2023-02-10)
+--------------------------
+
+- Fixes the dependency version specification by adding a missing comma.
```

Comment:
Fixes wcmatch dependency version specification.

(So that we are more concrete about which dependency.)

(Also, saying how (via adding comma) is not really needed.)